StripePaymentSheet/StripePaymentSheetTests/PaymentSheet/PaymentSheetFormFactorySnapshotTest.swift

```diff
@@ -448,6 +448,37 @@ final class PaymentSheetFormFactorySnapshotTest: STPSnapshotTestCase {
         XCTAssertTrue(formElement.validationState.isValid)
     }
 
+    func testEPM_subtitle_billingDetailCollectionScenarios() {
+        // Define test cases
+        let testCases: [(addressMode: PaymentSheet.BillingDetailsCollectionConfiguration.AddressCollectionMode,
+                         disableBillingDetailCollection: Bool,
+                         identifier: String)] = [
+                            (.automatic, true, "address_auto_disable_true"),
+                            (.automatic, false, "address_auto_disable_false"),
+                            (.never, true, "address_never_disable_true"),
+                            (.never, false, "address_never_disable_false"),
+                         ]
+
+        for (addressMode, disableBillingDetailCollection, identifier) in testCases {
+            var configuration = PaymentSheet.Configuration()
+            configuration.billingDetailsCollectionConfiguration.name = .always
+            configuration.billingDetailsCollectionConfiguration.email = .always
+            configuration.billingDetailsCollectionConfiguration.phone = .always
+            configuration.billingDetailsCollectionConfiguration.address = addressMode
+
+            // Create factory and form element
+            let factory = factory(for: .card, configuration: configuration)
+            let formElement = factory.makeExternalPaymentMethodForm(
+                subtitle: "Pay now with BufoPay",
+                disableBillingDetailCollection: disableBillingDetailCollection
+            )
+
+            let view = formElement.view
+            view.autosizeHeight(width: 375)
+            STPSnapshotVerifyView(view, identifier: identifier)
```

Comment:
I didn't realize that you could have a STPSnapshotVerifyView in a forloop. interesting.